Checkpoint1/Checkpoint1.cs

```diff
@@ -1,12 +1,317 @@
+                {
+                    Console.WriteLine($"The value you inserted was higher than expected. ({upBound})");
+                }
+                else if(testNum < loBound)
+                {
+                    Console.WriteLine($"The value you inserted was lower than expected. ({loBound})");
+                }
+                else
+                {
+                    return testNum;
+                }
+            }
+        }
+
+        // A method that exists only to get an integer without causing exceptions 
+        public static int GetNumber()
+        {
+            while(true)
+            {      
+                if(!Int32.TryParse(Console.ReadLine().Trim(), out int output1))
+                {
+                    Console.WriteLine("Please insert only integers.");
+                }
+                else
+                {
+                    return output1;
+                }                  
+            }
+        }
+        
+        /////////////////////////////////////////////////////////
+        // Challenge 1
+        // Goals:
+        // COUNT how many numbers are BETWEEN 1 and 100 ARE DIVISIBLE by 3 WITHOUT REMAINDER.
+        // Display that number in the console.
+        // Return to Main() loop without breaking anything. (More an issue with challenge 5)
+
+        public static void ChallengeOne()
+        {
+            int counter = 0;
+
+            for (int i = 1; i <= 100; i++)
+            {
+                if(i % 3 == 0)
+                {
+                    counter++;
+                }
+
+            }
+            Console.Clear();
+            Console.WriteLine($"There are {counter} numbers between 1 and 100 that are divisible by 3.");
+            Console.WriteLine("Press any key to return to the Main Menu");
+            Console.ReadKey();
+        }
+
+
+        /////////////////////////////////////////////////////////
+        // Challenge 2
+        // Goals:
+        // Loop and ask user for a number, or if they'd rather exit.
+        // Find the sum of all the numbers the user inputted and display that sum in the console.
+
+        public static void ChallengeTwo()
+        {
+            Console.Clear();
+            Stack<int> sumQueue = new Stack<int>();
+            string exitInput = null;
+
+            do
+            {
+                Console.WriteLine("Insert an integer to be added.");
+                sumQueue.Push(GetNumber());
+                Console.WriteLine("Type Q if you'd like to stop adding numbers.");
+                exitInput = Console.ReadLine().ToLower();
+            } while(exitInput != "q");
+
+            PopLoop(sumQueue);
+
+            Console.ReadKey();
+        }
+
+        public static int PopLoop(Stack<int> sumQueue)
+        {
+            int sum = 0;
+            string sumLine = null;
+
+            while(sumQueue.Count > 0)
+            {
+                switch(sumQueue.Count)
+                {
+                    case 1:
```

Comment:
could you have done this with just a sumQueue.Pop()? do you really need the switch with peeks and pop at the end?